src/App.jsx

```diff
@@ -14,57 +15,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+function Good({ good, selectedGood, setSelectedGood }) {
+  return good === selectedGood ? (
+    <tr
+      data-cy="Good"
+      className={good === selectedGood ? 'has-background-success-light' : ''}
+      key={good}
+    >
+      <td>
+        <button
+          data-cy="RemoveButton"
+          type="button"
+          className="button is-info"
+          onClick={() => setSelectedGood('')}
+        >
+          -
+        </button>
+      </td>
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+      <td data-cy="GoodTitle" className="is-vcentered">
+        {good}
+      </td>
+    </tr>
+  ) : (
+    <tr data-cy="Good" key={good}>
+      <td>
+        <button
+          data-cy="AddButton"
+          type="button"
+          className="button"
+          onClick={() => setSelectedGood(good)}
+        >
+          +
+        </button>
+      </td>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <td data-cy="GoodTitle" className="is-vcentered">
+        {good}
+      </td>
+    </tr>
```

Comment:
a lot of duplicates, you can combine it into the one button and use conditions where needed